src/analyzer.ts

```diff
@@ -363,7 +363,11 @@ export class Analyzer {
     if (
       wireitConfig !== undefined &&
       scriptCommand.value !== 'wireit' &&
-      scriptCommand.value !== 'yarn run -TB wireit'
+      scriptCommand.value !== 'yarn run -TB wireit' &&
+      // This form is useful when using package managers like yarn or pnpm which
+      // do not automatically add all parent directory `node_modules/.bin`
+      // folders to PATH.
+      !/^(..\/)+node_modules\/\.bin\/wireit$/.test(scriptCommand.value)
```

Comment:
Is there any test for this?

Reply:
Yep, see https://github.com/google/wireit/blob/1ff4a0ea89944447d8bcb99eb4916313095004bc/src/test/freshness.test.ts#L1527